diagnostics/model_test/template.json

```diff
@@ -2,13 +2,14 @@
     "model_name": "name_of_your_model",
     "reference_backend": "your_reference_backend",
     "testing_backend": "your_testing_backend",
-    "frozen_graph_location": "/path/to/the/frozen_graph.pb",
+    "graph_location": "/path/to/the/frozen_graph.pb",
```

Comment:
Maybe provide two template.jsons, one is template_pb.json and template_ckpt.json. I feel like it will be confusing if the template itself has two fields at the same time.

Reply:
good point, will do that.